.github/workflows/ci.yml

```diff
@@ -21,16 +21,21 @@ on:
       environment:
         type: string
         required: true
+      sign:
+        type: boolean
+        default: true
```

Comment:
If this defaults true, will our regular pr CI builds attempt to sign?

Reply:
I guess not as evidenced by this PR. 😄

Reply:
Yeah, regular CI builds gets triggered by the `push` and `pull_request` triggers which doesn't define this input

Reply:
Ah. Thanks for the explanation.